### PR TITLE
LPS Algorithm is Added Fixed Issues #746

### PR DESCRIPTION
**Pull request by karangadani**
This is my first GitHub contribution. I hope my pull request is added. 

Fixed Issue - #746

### CODE REVIEW

**Comment by ericklarac** [comment]
Add test cases for your implementation, please.

**Comment by karangadani** [comment]
@ericklarac  I'm really sorry, I am just a beginner I don't know much about test cases.

**Comment by ericklarac** [comment]
@karangadani no worries. Take this [PR](https://github.com/keon/algorithms/pull/765/files) as an example, adding test cases to your algorithm will help you confirm that your algorithm works for several inputs.

**Comment by karangadani** [comment]
@ericklarac  I learned something from the example and tried to implement test case. Let me know if I am doing it correct..

**Comment by ericklarac** [comment]
@karangadani lgtm

**Review by ericklarac — APPROVED** [review]

File: algorithms/map/longest_palindromic_subsequence.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    n = len(str)
+  
+    # Create a table to store results of subproblems
+    L = [[0 for x in range(n)] for x in range(n)]
+  
+    for i in range(n):
+        L[i][i] = 1
+  
+    
+    for cl in range(2, n + 1):
```
[the remark this answers is not in the document]
Comment (karangadani) [reply algorithms/map/longest_palindromic_subsequence.py:11]: C1 is length of Substring

[the remark this answers is not in the document]
Comment (ericklarac) [reply algorithms/map/longest_palindromic_subsequence.py:11]: Rename please

**Comment by karangadani** [comment]
@ericklarac when will my PR merge ? To be honest , as this is my first PR so Im too excited for it.

File: algorithms/map/longest_palindromic_subsequence.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (karangadani) [reply algorithms/map/longest_palindromic_subsequence.py:11]: Sure

**Pull request merged**